apps/full-site-editing/bin/docker-compose.override.yml.template

```diff
@@ -0,0 +1,13 @@
+services:
```

Comment:
nit - can dry this out a little. Depending on your taste for abstracting, with or without a base service:

```
services:
    wordpress-develop:
        &fse-base-service
        volumes:
           - ../full-site-editing-plugin:/var/www/${LOCAL_DIR-src}/wp-content/plugins/%PLUGIN_INSTALL_DIR%

    php:
        <<: *fse-base-service

    cli:
        <<: *fse-base-service

    # etc...
```

Reply:
Woah!!! That's pretty cool. Will add that :)

Reply:
This is super helpful too, because we might need to add more volumes for the unit testing stuff